code/modules/tables/rack.dm

```diff
@@ -24,3 +24,38 @@
 /obj/structure/table/rack/holorack/dismantle(obj/item/weapon/tool/wrench/W, mob/user)
 	to_chat(user, "<span class='warning'>You cannot dismantle \the [src].</span>")
 	return
+
+/obj/structure/table/rack/steel
+	color = "#666666"
+
+/obj/structure/table/rack/steel/New()
+	material = get_material_by_name(MAT_STEEL)

```

Comment:
This does seem odd, but its consistent with the existing tables in tables\presets.dm would you rather all of those be changed also?

Reply:
I'm going to go ahead and merge this one since it's consistent, but I think removing New() overrides that just set material directly should be a priority for a future PR.